backend/db/ddl_deprecated_statements.jinja.sql

```diff
@@ -1,4 +0,0 @@
--- Deprecated ----------------------------------------------------------------------------------------------------------
```

Comment:
## Renumbered files
Following removal of the `csets_to_ignore` DDL, I renumbered the files. However I think probably the numbering is not needed anymore now that we have this dependency graph. Added it to my misc todo list / longtail for DB refresh to see if I can remove the numbering.

I also removed an unneeded deprecated DDL file.